### ISSUE TITLE
Change the default height of input & select components to `$row-height-medium`

### ISSUE THREAD

**Issue by ramtinabadi** [Vieolo/vieolo-ui#316]
The default height should be turned to `$row-height-medium`.
This increase in height has the following advantages:

- Improves the interactivity for touch screens
- Adds the ability add buttons and other components inside the input/select

There should be an option to change the hight of the component as well.

The following components should be changed:
- [ ] Input and its derivatives
- [ ] Select
- [ ] DatePicker
